Readability.js

```diff
@@ -1446,7 +1446,7 @@ Readability.prototype = {
       if (elementProperty) {
         matches = elementProperty.match(propertyPattern);
         if (matches) {
-          for (var i = matches.length - 1; i >= 0; i--) {
+          for (var i = 0; i < matches.length; i++) {
```

Comment:
This alters the outcome, not just readability, right? `propertyPattern`'s `match` will have generated an array, and the loop overwrites `name` and `values[name]` with the last item in that array, instead of the first. Is that an intentional change? Should we just get the first item out of the list and then stop iterating?

Reply:
Yes, it actually changes the outcome. I seemed to have mindlessly changed any loop that was in reverse to loop in order. Getting the first element, which is the whole match is actually better.